tools/clang/tools/libclang/dxcisenseimpl.cpp

```diff
@@ -581,7 +581,9 @@ HRESULT DxcBasicUnsavedFile::Create(
   HRESULT hr = newValue->Initialize(fileName, contents, contentLength);
   if (FAILED(hr))
   {
-    delete newValue;
+    CComPtr<IMalloc> pTmp(newValue->m_pMalloc);
+    newValue->DxcBasicUnsavedFile::~DxcBasicUnsavedFile();
+    pTmp->Free(newValue);
```

Comment:
@pow2clck what'd happen if we just call `newValue->Release()` here? Same for `InternalDxcBlobEncoding_Impl`/`MemoryStream` above that had to get the `->T::~T` "fix" to get rid of the warning.

Reply:
Oh that's not entirely correct: Those two types have their code located in `Release()` already, and the code here also uses `IMalloc`. The only thing all three have in common: they use `IMalloc` (ie. not suitable for `DXC_MICROCOM_ADDREF_RELEASE_IMPL`) yet don't use `DxcThreadMalloc` (ie. not suitable for `DXC_MICROCOM_TM_ADDREF_RELEASE_IMPL` either).